Use six character width cyan rectangles for bottom row's buttons.

================================================================================
--- mc-4.6.2/src/widget.c
+++ mc-4.6.2/src/widget.c
@@ -2562,11 +2562,14 @@
 	attrset (DEFAULT_COLOR);
 	tty_printf ("%-*s", bb->widget.cols, "");
 	for (i = 0; i < COLS / 8 && i < 10; i++) {
+	    int j;
 	    widget_move (&bb->widget, 0, i * 8);
 	    attrset (DEFAULT_COLOR);
 	    tty_printf ("%d", i + 1);
 	    attrset (SELECTED_COLOR);
-	    tty_printf ("%-*s", ((i + 1) * 8 == COLS ? 5 : 6),
+	    j = columns_to_bytes(bb->labels [i].text ? bb->labels [i].text : "",
+	      ((i + 1) * 8 == COLS ? 5 : 6));
+	    tty_printf ("%-*s", j,
 		    bb->labels[i].text ? bb->labels[i].text : "");
 	    attrset (DEFAULT_COLOR);
 	}
